tasks/travis-pr-title-lint.js

```diff
@@ -73,7 +101,12 @@ module.exports = function(grunt) {
         });
         response.on('end', function() {
           try {
-            resolve(JSON.parse(body).title);
+            var prData = JSON.parse(body);
+            if (process.env.VERBOSE === 'true') {
+              console.log('DEBUG getPullRequestTitle:',     // eslint-disable-line no-console
```

Comment:
During these tests I've got again an empty pull request title once, but now the check fails with the expected error message ("unable to lint an empty commit message"), nevertheless I'm curious about the actual content of the result the we get from the github API, and so I've added some optionally verbose logging here.